armi/nuclearDataIO/xsCollections.py

```diff
@@ -785,13 +784,44 @@ def computeMacroscopicGroupConstants(
     multConstant=None,
     multLib=None,
 ):
-    """
+    r"""
     Compute any macroscopic group constants given number densities and a microscopic library.
 
     .. impl:: Compute macroscopic cross sections from microscopic cross sections and number densities.
         :id: I_ARMI_NUCDATA_MACRO
         :implements: R_ARMI_NUCDATA_MACRO
 
+        This method computes the macroscopic cross sections of a specified
+        reaction type from inputted microscopic cross sections and number
+        densities. The ``constantName`` parameter specifies what type of
+        reaction is requested. The ``numberDensities`` parameter is dictionary
+        mapping the nuclide to its density. The ``lib`` parameter is a library
+        object like :py:class:`~armi.nuclearDataIO.xsLibraries.IsotxsLibrary` or
+        :py:class:`~armi.nuclearDataIO.xsLibraries.CompxsLibrary` that holds the
+        microscopic cross-section data. The ``microSuffix`` parameter specifies
+        from which part of the library the microscopic cross sections are
+        gathered; this is typically gathered from a components
+        ``getMicroSuffix`` method like :py:meth:`Block.getMicroSuffix
+        <armi.reactor.blocks.Block.getMicroSuffix>`. ``libType`` is an optional
+        parameter specifying whether the reaction is for neutrons or gammas.
+        This method also has the optional parameters ``multConstant`` and
+        ``multLib``, which allows another reaction's cross section to be
```

Comment:
I don't think this is quite right. These allow for a scalar multiplier to be extracted from multLib, such as nu, energy_per_fission, etc.